packages/paste-website/src/pages/components/blockquote/index.mdx

```diff
@@ -0,0 +1,192 @@
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote>
+      <BlockquoteContent>
+        If I must die <br/>
+        let it bring hope <br/>
+        let it be a tale
+      </BlockquoteContent>
+      <BlockquoteCitation author="Rafaat Alareer" source="If I Must Die" />
+    </Blockquote>`}
+</LivePreview>
+
+### Blockquote with author only
+
+Use a Blockquote with only author attribution when you want to highlight a quote from an external source, but there is no specific need for source or the source is unavailable.
+
+Example use cases:
+- When quoting a well-known individual, where the author’s name provides enough context for the audience.
+- When quoting from a source that is hard to trace or not directly accessible (e.g., a verbal interview or a widely repeated saying without a specific source)
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote>
+      <BlockquoteContent>
+        When you light a candle, you also cast a shadow.
+      </BlockquoteContent>
+      <BlockquoteCitation author="Ursula K. Le Guin" />
+    </Blockquote>`}
+</LivePreview>
+
+## Composition notes
+
+- Whenever possible, include a URL to the original source to allow users to access additional context or details
+- Always provide context before and after the Blockquote to connect the information to the surrounding content
+- Use **capital case** for author’s name. For example: Jane Doe
+- Use **sentence case**, with only the first word and proper nouns capitalized for the source and its URL
+- Blockquote should maintain a vertical spacing of `$space-70` from its surrounding content
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation, Paragraph}}
+  language="jsx"
+>
+  {`<>
+    <Paragraph>Avoid being too heavy-handed with communicating how a feature has been built or changed with AI since introducing new interaction patterns and unnecessary messaging can distract from the customer’s goals.</Paragraph>
+      <Blockquote url="#">
+        <BlockquoteContent>
+          With AI-driven products, there can be a temptation to communicate the “newness” or “magic” of the system’s predictions through its UI metaphors. However, unfamiliar UI touchpoints can make it harder for users to learn to use your system, potentially leading to degraded understanding of, or trust in, your product…Instead, anchor new users with familiar UI patterns and features.
+        </BlockquoteContent>
+        <BlockquoteCitation author="Google" source="People + AI Guidebook" />
+      </Blockquote>
+      <Paragraph marginBottom="space0">With AI features, there’s also a higher chance for things to go wrong.</Paragraph>
+    </>`}
+</LivePreview>
+
+
+## Dos and Don'ts
+
+<DoDont>
+  <Do title="Do" body="Maintain the integrity of the external reference and refrain from altering its meaning." />
```

Comment:
Question: Do you think we should include a visual example for the first Dos and Don't section?

Reply:
I left it out because they seemed straightforward enough but happy to add if you want them in there! LMK

Reply:
I agree, we can skip on it!